src/main.ts

```diff
@@ -95,12 +95,53 @@ export function isSuccess<E, A>(monadA: Dataway<E, A>): monadA is Success<A> {
   return monadA._tag === 'Success';
 }
 
+/**
+ * Construct a new `Dataway` from a nullable type, if the value is `null` or `undefined`,
+ * returns `NotAsked` else returns the value wrapped in a `Success`
```

Comment:
```suggestion
 * returns `NotAsked` else returns the value wrapped in a `Success`
```
```
Constructs a new `Dataway` from a nullable type. 
If the value is `null` or `undefined` it returns `NotAsked`,
otherwise it returns the value wrapped in a `Success`
```